README.md

```diff
@@ -36,6 +36,45 @@ Install package via PyPI
 pip3 install dxfeed
 ``` 
 
+## Installation from sources
+
+Reminder: initialize and pull git submodule after cloning the repo:
+```bash
+git submodule init
+git submodule update
+``` 
+
+To install dxfeed from source you need Poetry, Cython and taskipy in addition to the normal dependencies above.
```

Comment:
Seems to be, that I don't have Cython and taskipy installed outside of the project's virtualenv. So if we are talking about global dependencies - this two are redundant, but if it's about internal ones - this list seems to be too short, isn't it? 🤔